frontend/src/main/web/src/components/TableRow.js

```diff
@@ -0,0 +1,57 @@
+import React, { PropTypes } from 'react'
+import { merge } from 'lodash'
+import { Row } from './'
+
+const classes = {
+  base: {
+    bd: 'Bdb(bd1) Bdc(light)',
+    h: 'H(r2)'
+  },
+  highlight: {
+    trs: 'Trs(aeo)',
+    hover: {
+      bgc: 'Bgc(light):h'
+    }
+  },
+  selected: {
+    bgc: 'Bgc(light)',
+    custom: 'row--selected'
+  }
+}
+
+const TableRow = ({
```

Comment:
Needs doc comment. Probably should mention using `TableCell` with it, and any other component this should usually be used with (do you need to wrap it in some table component or something, or can you just use these with no wrapper?)

Maybe refer to a component/container where there is an example of it being used, if there is one that is clear to read.